android/sdk/src/main/java/com/taobao/weex/utils/FontDO.java

```diff
@@ -73,7 +73,7 @@ private void parseSrc(String src, WXSDKInstance instance) {
       return;
     }
 
-    if (src.matches("^url\\('.*'\\)$")) {
+    if (src.matches("^url\\((('[^']*')|(\"[^\"]*\"))\\)$")) {
```

Comment:
Why not use `^url\\((('.*')|(\".*\"))\\)$` ?

Reply:
My idea is that no quotes are allowed inside http url.
Maybe you are right, without this restriction will get better compatibility.
Would you like me to change the code?